app/vmalert/rule/group.go

```diff
@@ -471,6 +471,7 @@ func (g *Group) DeepCopy() *Group {
 	newG := Group{}
 	_ = json.Unmarshal(data, &newG)
 	newG.Rules = g.Rules
+	newG.id = g.id
```

Comment:
@Haleygo maybe it would be more straightforward to rm `func (g *Group) GetID()` and make `id` public? What's the benefit of having getter here?

Reply:
group.id was changed to private since it shouldn't be modified after creation.
I prefer this fix here over making `id` public.